### PR TITLE
ng2-translate integrated and refactored the e2e tests; BUG fixed

### PR DESCRIPTION
**Pull request by despotovska**
- refactored the home.html and userSettings.html;
- refactored the e2e tests and page objects and added new one;
- implemented ng2-translate modified all the html-s/components/services that had hardcoded macedonian words;
- added validations for username field on register page and for uploading invalid format file;

### CODE REVIEW

File: e2e/RegisterTest.e2e.js [hunk 1]
```diff
@@ -1,57 +1,58 @@
...
+
   var LogInPage = require("./pages/LoginPageObject.js");
+  var RegisterPage = require("./pages/RegisterPageObject.js");
 
   var loginUrl = "http://localhost:3000/#/login";
   var usernameJosif = "Josif";
   var successMessage = "Успешно внесен корисник.";
   var existingUsernameMessage = "Корисничкото име веќе постои, обидете се да се регистрирате со друго име.";
-  var selectPictureMessage = "За да креирате профил, ве молам изберете слика.";
+  var selectPictureMessage = "За да креирате профил, мора да изберете слика.";
```
Comment (eljesaz) [inline e2e/RegisterTest.e2e.js:10]: llajk za ova izmena ;) (Y)

**Comment by eljesaz** [comment]
+1

**Comment by ebarabanovska** [comment]
+1

**Comment by dnoveska** [comment]
+1

**Pull request merged**